app/jobs/process_delius_data_job.rb

```diff
@@ -24,76 +24,78 @@ def perform(nomis_offender_id)
 private
 
   def import_data(nomis_offender_id)
-    delius_record = OffenderService.get_community_data(nomis_offender_id)
-    delius_com_info = OffenderService.get_com(nomis_offender_id)
-    offender = OffenderService.get_offender(delius_record['noms_no'])
+    probation_record = OffenderService.get_probation_record(nomis_offender_id)
 
-    if offender.nil?
-      return logger.error("[DELIUS] Failed to retrieve NOMIS record #{delius_record['noms_no']}")
-    end
+    return logger.error("[DELIUS] Failed to retrieve probation record for #{nomis_offender_id}") if probation_record.nil?
```

Comment:
Can we have some lovely log-event based logging?

https://dsdmoj.atlassian.net/wiki/spaces/OCM/pages/4187389997/MPC+Logging+Standards